Cargo.toml

```diff
@@ -14,6 +14,9 @@ anyhow = "1.0.12"
 clap = { version = "2.33", default-features = false }
 liboverdrop = "0.0.2"
 rust-ini = ">=0.13, <0.16"
+kernlog = { git = "https://github.com/nabijaczleweli/kernlog.rs.git", rev = "a44a3f021815e091966741aa3bcdcefdfd51590d" }
```

Comment:
I'm not very comfortable with this because it would mean we are not able to make a release in Fedora...

Reply:
How about a submodule or a subtree?

Reply:
Not really, we need it on crates.io

Reply:
The only options, as I see it, are:
  * speccing a github repository (as it is now),
  * convincing the kernlog maintainer to do a bunch of work (merging the PR, testing, publishing, the lot), or
  * vendoring the thing.

The first one is understandably problematic, the second is either not possible or a bit scummy, I can't really see why the third wouldn't work, unless I'm missing something obvious; it'd potentially duplicate kernlog (though, in reality, kernlog doesn't actually build nowadays), but that's hardly RC?

Reply:
Let's wait for @kstep's response. If the project is dead then my preference would be to simply integrate the code locally. It's ~120 lines of code, and we could probably get rid of the simplelog fallback. So maybe that's the thing to do anyway?

Reply:
Fully agree, or @nabijaczleweli can just create a new crate that will not do a fallback and we can use that.

Reply:
>and we could probably get rid of the simplelog fallback

Oh I think I haven't quite explained clearly enough.

As I've noted at the top (reproduced here in full):
1. The maxi approach taken here, using another logger (here: `simplelog`, which is rather large dependency-wise), if kernlog doesn't initialise.
2. The midi approach, using, for example, a "develop" feature, which will enable `simplelog` only for builds you specify (I think one could be devilish and enable it only on the debug build profile, but).
3. The mini approach: simply not. If the logger initialises, great; if not: whatever. If no `log` logger is installed, the output is ignored.
4. The micro approach: `kernlog::init().unwrap()` or equivalent.

Using `simplelog` is just one potential approach that I continued with since nobody objected. Dropping `simplelog` is as trivial as removing the dependency line from `Cargo.toml` and one init line in `main.rs`. To reiterate: it is *not* a property of `kernlog` *at all*.
IOW what you're proposing is simply switching to the mini approach.

I can integrate kernlog into here if you want, but I still think it'd be a better idea to vendor it, so I'll hold off until we're all on the same page.